warehouse-management-service/pkg/database/postgres/postgres.go

```diff
@@ -31,6 +29,11 @@ func New(config config.Postgres) (*Postgres, error) {
 	}, nil
 }
 
+func Connection(config config.Postgres) string {
+	return fmt.Sprintf("host=%s port=%s user=%s password=%s dbname=%s sslmode=%v",
+		config.Host, config.Port, config.Username, config.Password, config.DBName, config.SSLMode)
```

Comment:
I'm not sure where this is printing to, but in general, its a good idea to not expose these things. Don't want them to get picked up by a monitoring tool or a log-rotater for example, and exposing your credentials that way.

Reply:
`Sprintf` formats and returns a string, it doesn't print it.

Separately, I've extracted creating a connection string out to a new function because it is now used in 2 places, to connect to the database and to run migrations